src/pages/[platform]/build-a-backend/data/connect-event-api/index.mdx

```diff
@@ -283,3 +533,307 @@ export default function App() {
   );
 }
 ```
+</InlineFilter>
+
+<InlineFilter filters={['android']}>
+## Client Library Usage Guide
+
+### Create the Events class
+
+You can find your endpoint in the AWS AppSync Events console. It should start with `https` and end with `/event`.
+
+```kotlin
+val endpoint = "https://abcdefghijklmnopqrstuvwxyz.appsync-api.us-east-1.amazonaws.com/event"
+val events = Events(endpoint)
+```
+
+### Using the REST Client
+An `EventsRestClient` can be created to publish event(s) over REST. It accepts a publish authorizer that will be used by default for any publish calls within the client.
+
+#### Creating the REST Client
+``` kotlin
+val events: Events // Your configured Events
+val restClient = events.createRestClient(
+  publishAuthorizer = ApiKeyAuthorizer("da2-abcdefghijklmnopqrstuvwxyz")
+)
+```
+
+Additionally, you can pass custom options to the Rest Client. Current capabilities include modifying the `OkHttp.Builder` the `EventsRestClient` uses, and enabling client library logs.
+See [Collecting Client Library Logs](#collecting-client-library-logs) for the AndroidLogger class.
+
+``` kotlin
+val restClient = events.createRestClient(
+    publishAuthorizer = ApiKeyAuthorizer("da2-abcdefghijklmnopqrstuvwxyz"),
+    options = Events.Options.Rest(
+        loggerProvider = { namespace -> AndroidLogger(namespace, logLevel) },
+        okHttpConfigurationProvider = { okHttpBuilder ->
+            // update OkHttp.Builder used by EventsRestClient
+        }
+    )
+)
+```
+
+#### Publish a Single Event
+```kotlin
+val restClient: EventsRestClient // Your configured EventsRestClient
+// List of kotlinx.serialization.json.[JsonElement, JsonPrimitive, JsonArray, JsonObject]
+val jsonEvent = JsonObject(mapOf("some" to JsonPrimitive("data")))
+val publishResult = restClient.publish(
+  channelName = "default/channel", 
+  event = jsonEvent
+)
+when(publishResult) {
+  is PublishResult.Response -> {
+    publishResult.successfulEvents // inspect successful events
+    publishResult.failedEvents // inspect failed events
```

Comment:
Here (and elsewhere) it might be better to say

```suggestion
    val successfulEvents = publishResult.successfulEvents // inspect successful events
    val failedEvents = publishResult.failedEvents // inspect failed events
```

Otherwise it just kind of looks weird to have the property accessors not used for anything.